### PR TITLE
HAWQ-1263 - misc doc updates

### PR DESCRIPTION
**Pull request by lisakowen**
this PR includes:

- AWS configuration document updates including clarifying storage options and security groups
- minor gpfdist example update for single instance

### CODE REVIEW

File: markdown/ddl/ddl-database.html.md.erb [hunk 1]
```diff
@@ -2,77 +2,75 @@
...
 
 ``` sql
-=> SELECT datname FROM pg_database;
+=# SELECT datname FROM pg_database;
 ```
 
 ## <a id="topic7"></a>Altering a Database 
 
-The ALTER DATABASE command changes database attributes such as owner, name, or default configuration attributes. For example, the following command alters a database by setting its default schema search path \(the `search_path` configuration parameter\):
+Use the [ALTER DATABASE](../reference/sql/ALTER-DATABASE.html) command to update database attributes including owner or name. You can also use ALTER DATABASE to update a database's default configuration. For example, the following command alters a database by setting its default schema search path using the `search_path` server configuration parameter:
```
Comment (dyozie) [inline markdown/ddl/ddl-database.html.md.erb]: "For example, the following command alters a database by setting its default schema search path using the `search_path` server configuration parameter:" -> "For example, the following command alters the `search_path` server configuration parameter to change the default schema search path:"

File: markdown/ddl/ddl-database.html.md.erb [hunk 2]
```diff
@@ -2,77 +2,75 @@
...
-=> ALTER DATABASE mydatabase SET search_path TO myschema, public, pg_catalog;
+=# ALTER DATABASE mydatabase SET search_path TO myschema, public, pg_catalog;
 ```
 
 To alter a database, you must be the owner of the database or a superuser.
 
 ## <a id="topic8"></a>Dropping a Database 
 
-The `DROP DATABASE` command drops \(or deletes\) a database. It removes the system catalog entries for the database and deletes the database directory on disk that contains the data. You must be the database owner or a superuser to drop a database, and you cannot drop a database while you or anyone else is connected to it. Connect to `template1` \(or another database\) before dropping a database. For example:
+The [DROP DATABASE](../reference/sql/DROP-DATABASE.html) command drops \(or deletes\) a database. It removes the system catalog entries for the database and deletes the data on disk. You must be the database owner or a superuser to drop a database, and you cannot drop a database while you or anyone else is connected to it. Connect to `template1` \(or another database\) before dropping a database. For example:
```
Comment (dyozie) [inline markdown/ddl/ddl-database.html.md.erb]: (or deletes) -> (deletes)

File: markdown/ddl/ddl-database.html.md.erb [hunk 3]
```diff
@@ -2,77 +2,75 @@
...
-The host name and port must match the host name and port of the installed HAWQ system.
+The host name and port must match the host name and port of the HAWQ master node.
 
-Some objects, such as roles, are shared by all the databases in a HAWQ system. Other objects, such as tables that you create, are known only in the database in which you create them.
+Some objects, such as roles, are shared by all of the databases in a HAWQ deployment. Other objects, such as tables that you create, are known only to the database in which you create them.
 
 ### <a id="topic5"></a>Cloning a Database 
 
-By default, a new database is created by cloning the standard system database template, *template1*. Any database can be used as a template when creating a new database, thereby providing the capability to 'clone' or copy an existing database and all objects and data within that database. For example:
+By default, a new database is created by cloning the standard system database template, `template1`. You may identify an alternate template to use for database creation. As mentioned in the section above, you can specify `template0` as the template:
```
Comment (dyozie) [inline markdown/ddl/ddl-database.html.md.erb]: Also re: cloning, the CREATE DATABASE page has this wording, which seems to suggest a regular db can be named as a template.  Probably want to do some editing here as well:

"When you copy a database by specifying its name as the template, no other sessions can be connected to the template database while it is being copied. New connections to the template database are locked out until CREATE DATABASE completes."

File: markdown/ddl/ddl-database.html.md.erb [hunk 4]
```diff
@@ -2,77 +2,75 @@
...
+Each new database you create is based on a *template*. HAWQ provides a default database, `template1`. HAWQ uses `template1` to create databases unless you specify another template. Do not create any objects in `template1` unless you want those objects to be in every database you create.
 
-HAWQ uses two other database templates, *template0* and *postgres*, internally. Do not drop or modify *template0* or *postgres*. You can use *template0* to create a completely clean database containing only the standard objects predefined by HAWQ at initialization, especially if you modified *template1*.
+HAWQ uses two other database templates internally, `template0` and `postgres`. Do not drop or modify `template0` or `postgres`. You can use `template0` to create a completely clean database containing only the standard objects predefined by HAWQ at initialization, especially if you have modified `template1`.
```
Comment (lisakowen) [inline markdown/ddl/ddl-database.html.md.erb]: i think template0 and template1 start out the same.  you cannot modify template0, but you can template1.

**Comment by lisakowen** [comment]
PR merged; closing.

**Pull request closed without merging**